docs/custom-component.md

```diff
@@ -1,22 +1,24 @@
 Custom Components
 ---------
 
-In order to provide the flexibility necessary to build an advanced search experience, Vue Instant Search includes the ability to build custom components.
+You can build you own components when the provided ones are not sufficient. Since we try
```

Comment:
@ehayman did an iteration on my text which was a lot more verbose.
She probably didn't want to remove some of my stuff.
I think your version looks good as well.
Let's wait her feedback.